### ISSUE TITLE
Minimize to Tray

### ISSUE THREAD

**Issue by Airyzz** [commetchat/commet#72]

**Comment by ollietheotter983** [comment]
this exists now doesnt it? having the x button minimise the app

**Comment by Airyzz** [comment]
minimizing is not the same as minimizing to the tray

**Comment by byjove01** [comment]
I agree. Not having a sys tray feature makes the app look too amateur. I mean, it is, but... yeah, you probably get what I mean.

**Comment by Dark-Sheepa** [comment]
That was the first thing about Windows version that disappointed me, and I had to get used to it over many days. 😂
